### ISSUE TITLE
Error In Demo

### ISSUE THREAD

**Issue by ComputerASCII** [c-as/godot-voip#13]
<img width="1675" alt="Screen Shot 2021-05-16 at 1 52 02 PM" src="https://user-images.githubusercontent.com/77028901/118409001-f8edc080-b64d-11eb-99f8-6c1e1a7c9dfc.png">
I downloaded it straight from your latest release on GitHub. I wouldn't be surprised if I'm just doing it wrong, but I would certainly appreciate your help troubleshooting this. I'm using Godot version 3.3 stable. I read and attempted to follow the readme.

**Comment by ikbencasdoei** [comment]
Can you tell when the errors start appearing exactly? Also are there things that still work? I assume you didn't make any changes.

**Comment by ikbencasdoei** [comment]
Also, what platform are you on?

**Comment by ComputerASCII** [comment]
> Can you tell when the errors start appearing exactly? Also are there things that still work? I assume you didn't make any changes.

The error occurs when I click start server and then listen. The errors continue to stack up until the program is closed. When I click start client before pressing start server, it says "Status: Failed to connect to server!". I did not mess with the default settings.

>Also, what platform are you on?

I'm on Macintosh, but I also have a PC (which I have not tried this on yet)

**Comment by ikbencasdoei** [comment]
Do the errors also occur when starting a server and then speaking but with listening disabled?

**Comment by ComputerASCII** [comment]
<img width="1672" alt="Screen Shot 2021-05-17 at 12 48 26 PM" src="https://user-images.githubusercontent.com/77028901/118533890-39ad0e80-b70e-11eb-9ba2-7f8c5e96053c.png">
Yes.
<img width="1674" alt="Screen Shot 2021-05-17 at 12 51 32 PM" src="https://user-images.githubusercontent.com/77028901/118534284-aa542b00-b70e-11eb-8783-d806121f32cc.png">
This only pops up the first time you install it so I didn't get a screenshot at first, but it's definitely part of the problem.

> Unable to load addon script from path: 'res://addons/godot-voip/plugin.gd' There seems to be an error in the code, please check the syntax.

**Comment by ikbencasdoei** [comment]
Thank you for showing me the popup. I fixed it but it is not related.

I just found this issue with the same problem: https://github.com/godotengine/godot/issues/35445. Someone fixed it by setting the mix_rate in the project-settings to match the mix_rate setting in macOS.

**Comment by ComputerASCII** [comment]
>Thank you for showing me the popup. I fixed it but it is not related.

Alright lol, shows how much I need to learn. Glad I could help.
 >I just found this issue with the same problem: godotengine/godot#35445. Someone fixed it by setting the mix_rate in the project-settings to match the mix_rate setting in macOS.
<img width="1678" alt="Screen Shot 2021-05-17 at 3 22 07 PM" src="https://user-images.githubusercontent.com/77028901/118552144-2ad15680-b724-11eb-866f-1d622a4e5bc7.png">
That worked like a charm, all I had to do was change the format to 44,100 Hz in Audio MIDI setup. Closing this issue, thanks for helping and for making this add-on.
